doc/api.md

```diff
@@ -742,6 +744,58 @@ Unreserves a path for a publishing application. Returns success or failure only.
 - If a path reservation exists for the supplied base_path but a different
   publishing_app, the command will fail.
 
+## `PUT /publish-intent(/*base_path)`
+
+[Request/Response detail][put-intent-pact]
+
+Used to create or update a publishing intent for a particular base_path and
+reserve the base path in the publishing-api. A publishing intent sets when the
+cache should expire for a piece of content.
+
+### Path parameters
+
+- `base_path`
+  - Identifies the path to create the publishing intent for
+
+### JSON parameters:
+
+- `publishing_app` *(required)*
+  - The name of the application making this request.
+- `rendering_app` *(required)*
+  - The name of the application rendering the content the base path refers to.
+- `publish_time` *(required)*
+  - The `DateTime` the content is set to be published.
+- `routes` *(optional)*
+  - `Array` - Fields supported are path and type, the exact requirements can be
+    found [here][routes-content-schema]. This is optional as it is only
+    required if there needs to be a route created in router-api for the
```

Comment:
Minor one, but would be helpful to indicate this is done immediately ;-).

Reply:
Yep sure, i've reworded the sentence to make it clearer that this is done when the publishing intent is made. I steered clear of using the word `immediately` as to me this denotes that that if it isn't done immediately, it will be done asynchronously.